src/services/user_api/default.txt

```diff
@@ -0,0 +1,10 @@
+RF: improving code of the ft-user-api
```

Comment:
remove this default.txt file as all the messages should be in your commit message